reminders/models.py

```diff
@@ -0,0 +1,48 @@
+from django.core.exceptions import ValidationError
+from django.utils import timezone
+from django.db import models
+from events.models import EventParticipant
+
+
+def validate_date(date_time):
+    if timezone.now() > date_time:
+        raise ValidationError(f'{time_format(date_time)} is not a valid date_time')
+
+
+def time_format(date_time):
+    return date_time.strftime("%Y-%m-%d %H:%M:%S")
+
+
+class Reminder(models.Model):
+    class EMethods(models.TextChoices):
+        WEBSITE = "web", "Website"
+        EMAIL = "ema", "Email"
+        WEBSITE_EMAIL = "wae", "Website and Email"
+
+    participant_id = models.ForeignKey(EventParticipant, on_delete=models.CASCADE)
+    date_time = models.DateTimeField(default=timezone.now, validators=[validate_date])
+    messages = models.TextField(null=True, blank=True)
+
+    method = models.CharField(
+        max_length=3,
+        choices=EMethods.choices,
+        default=EMethods.WEBSITE,
+    )
+
+    def validate_unique_reminder(self, participant, date_time):
+        if Reminder.objects.filter(participant_id__user_id=participant.user_id,
+                                   participant_id__event_id=participant.event_id,
+                                   date_time=date_time):
```

Comment:
usually the convention is like:
```suggestion
        if Reminder.objects.filter(
                participant_id__user_id=participant.user_id,
                participant_id__event_id=participant.event_id,
                date_time=date_time
        ):
```